src/platform/plugins/shared/home/server/services/tutorials/tutorials_registry.ts

```diff
@@ -74,7 +74,10 @@ export class TutorialsRegistry {
   private readonly scopedTutorialContextFactories: TutorialContextFactory[] = [];
   private staticAssets!: IStaticAssets;
 
-  constructor(private readonly initContext: PluginInitializerContext) {}
+  constructor(
+    private readonly initContext: PluginInitializerContext,
+    private readonly isServerless = false
```

Comment:
Since the full initContext is available here, it seems like we could get the isServerless from there?